ompi/mpi/c/bsend_init.c

```diff
@@ -79,6 +80,13 @@ int MPI_Bsend_init(const void *buf, int count, MPI_Datatype type,
         OMPI_ERRHANDLER_RETURN(rc, comm, rc, FUNC_NAME);
     }
 
+#if OPAL_ENABLE_FT_MPI
+    /*
+     * The request will check checked for process failure errors during the
```

Comment:
typo: will be checked

-> This is only a comment added, if FT_MPI is on?

Reply:
why not? by spec we check all errors at completion call, not initation. That would mean returning an error at WAIT, not INIT or START;

Reply:
I think we both tried to have a consistent approach across similar functions, but our definition of "similar" differs. I see that all persistent request are consistent among them, but they differ from their blocking and non-blocking counterparts. I would have made the cut for "similar" at the functionality level, such that all flavors of send have the same behavior.

In particular for persistent communications, requests created before a fault will return the corresponding error from WAIT and/or START, but why should we allow the creation of a persistent request with a peer that is known as dead ?

However, I don't think this is such a big issue right now, we can address it at a later stage.